src/utilities/input.tsx

```diff
@@ -0,0 +1,41 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement } from 'react';
+
+export enum HelperTextStyle {
```

Comment:
Union types: `'success' | 'danger'` instead of `enum Style { Success, Danger }`

Reply:
There are some validation type missing. You could also use what we have in `utilities/validation.ts`
```
export enum Validation {
    Default = 'Default',
    Loading = 'Loading',
    Success = 'Success',
    Error = 'Error',
    Warning = 'Warning',
}

export const validationClassMap: Record<Validation, string> = {
    [Validation.Default]: 'tw-border-line-strong',
    [Validation.Loading]: 'tw-border-line',
    [Validation.Success]: 'tw-border-text-positive',
    [Validation.Error]: 'tw-border-text-negative',
    [Validation.Warning]: 'tw-border-text-warning',
};
```

You could add text-color there and make sure to use alias-tokens.

Reply:
Since it was a new helper I kept the attributes to a minimum, what the new component needed.  I can either update it with all of these for potential future use or switch the component to use the this version of the text color handling.

Reply:
The new component needs all validation to be handled because they are already in use in the web-app. The Figma file doesn't all of them but you can see that the current implementation of the `TextInput`,`Textarea`, `Dropdown`, `MultiSelect` have a property `validation` with the styling already in place.